<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_widget</class>
 <widget class="QMainWindow" name="main_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>958</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Overmix</string>
  </property>
  <widget class="QWidget" name="central_layout">
   <layout class="QHBoxLayout" name="main_layout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tab_pages">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_preview">
       <attribute name="title">
        <string>Preview</string>
       </attribute>
       <layout class="QHBoxLayout" name="preview_layout">
        <item>
         <layout class="QVBoxLayout" name="properties_layout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="lbl_info"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="comparing_group">
            <property name="title">
             <string>Comparing</string>
            </property>
            <layout class="QVBoxLayout" name="comparing_layout">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="merge_group">
            <property name="title">
             <string>Aligning</string>
            </property>
            <layout class="QVBoxLayout" name="align_layout">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>3</number>
               </property>
               <item>
                <widget class="QCheckBox" name="cbx_each_frame">
                 <property name="text">
                  <string>For each frame</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_subpixel">
                 <property name="text">
                  <string>Align</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="render_group">
            <property name="title">
             <string>Render</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="render_layout">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="QCheckBox" name="render_redraw">
               <property name="text">
                <string>Redraw on changes</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="postprocess_group">
            <property name="title">
             <string>Postprocessing</string>
            </property>
            <layout class="QVBoxLayout" name="postprocess_layout">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="color_group">
            <property name="title">
             <string>Color</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <property name="spacing">
              <number>3</number>
             </property>
             <item row="0" column="1">
              <widget class="QCheckBox" name="cbx_nocolor">
               <property name="text">
                <string>Show planes</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="cbx_dither">
               <property name="text">
                <string>Dither</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="cbx_nogamma">
               <property name="text">
                <string>Disable gamma correction</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <widget class="QPushButton" name="btn_refresh">
              <property name="text">
               <string>Draw</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_clear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QPushButton" name="btn_save">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_as_mask">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Use as mask</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_files">
       <attribute name="title">
        <string>Files</string>
       </attribute>
       <layout class="QHBoxLayout" name="files_layout">
        <item>
         <layout class="QVBoxLayout" name="_2">
          <item>
           <widget class="QCheckBox" name="cbx_interlaced">
            <property name="text">
             <string>De-telecine images on input</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="preprocess_group">
            <property name="title">
             <string>Modify files</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="preprocess_layout">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="selection_group">
            <property name="title">
             <string>Selection</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QComboBox" name="selection_selector">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string>Everything</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Group</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Frame</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Custom</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_apply_mods">
               <property name="text">
                <string>Apply modifications</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="images_group">
            <property name="title">
             <string>Images</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QTreeView" name="files_view">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::ContiguousSelection</enum>
               </property>
               <attribute name="headerDefaultSectionSize">
                <number>40</number>
               </attribute>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QPushButton" name="btn_add_group">
                 <property name="text">
                  <string>Add group</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_delete_files">
                 <property name="text">
                  <string>Delete</string>
                 </property>
                 <property name="shortcut">
                  <string>Del</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_save_files">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="masks_group">
            <property name="title">
             <string>Alpha masks</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QTreeView" name="mask_view">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <attribute name="headerDefaultSectionSize">
                <number>60</number>
               </attribute>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QPushButton" name="pre_alpha_mask">
                 <property name="text">
                  <string>Add masks</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pre_remove_mask">
                 <property name="text">
                  <string>Remove</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pre_clear_mask">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Clear mask</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Fi&amp;les</string>
    </property>
    <addaction name="action_add_files"/>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>V&amp;iew</string>
    </property>
    <addaction name="action_show_menubar"/>
    <addaction name="action_fullscreen"/>
    <addaction name="action_movement_graph"/>
    <addaction name="action_make_black"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_online_wiki"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_crop_all"/>
    <addaction name="action_create_slide"/>
    <addaction name="action_show_skip_preview"/>
    <addaction name="action_create_sr"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuView"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_online_wiki">
   <property name="text">
    <string>&amp;Online wiki</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_fullscreen">
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="action_add_files">
   <property name="text">
    <string>&amp;Add files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_show_menubar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Show menubar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_movement_graph">
   <property name="text">
    <string>&amp;Display movement graph</string>
   </property>
  </action>
  <action name="action_crop_all">
   <property name="text">
    <string>&amp;Crop to area</string>
   </property>
  </action>
  <action name="action_create_slide">
   <property name="text">
    <string>Create &amp;slide</string>
   </property>
  </action>
  <action name="action_make_black">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Black background</string>
   </property>
  </action>
  <action name="action_show_skip_preview">
   <property name="text">
    <string>Show Demosaic Calculator</string>
   </property>
  </action>
  <action name="action_create_sr">
   <property name="text">
    <string>Create SR sample</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
